metadata/aquatics-2021-05-01-BTW.yml

```diff
@@ -157,18 +157,18 @@ metadata:
 #match the dimensionality of the initial_conditions unless there are state
 #variables where process error is not being estimated or propagated
     process_error:
-      status: absent #options: absent, present, data_driven, propagates, assimilates
-      complexity: 2 #Leave blank if status = absent
+      status: assimilates #options: absent, present, data_driven, propagates, assimilates
+      complexity: 1 #Leave blank if status = absent
       #Leave everything below blank if status = absent, present, or data_driven
       propagation:
         type: ensemble #How does your model propogate random effects uncertainty (ensemble or MCMC is most common)
-        size: 2000
+        size: 31
       #Leave everything below blank  UNLESS status = assimilates
       assimilation:
-        type: refit
-        reference: "none"
-        complexity: 4
-        covariance: FALSE #TRUE OR FALSE
+        type: EnKF
+        reference: https://aslopubs.onlinelibrary.wiley.com/doi/full/10.1002/lom3.10302
+        complexity: 1
```

Comment:
should be same size as the number of states we're updating, which is just water temperature